content/en/infrastructure/containers/configuration.md

```diff
@@ -103,6 +103,7 @@ The following table presents the list of collected resources and the minimal Age
 | ServiceAccounts | 7.27.0 | 1.19.0 | 2.30.9 | 1.17.0 |
 | Services | 7.27.0 | 1.11.0 | 2.10.0 | 1.17.0 |
 | Statefulsets | 7.27.0 | 1.15.0 | 2.20.1 | 1.16.0 |
+| VerticalPodAutoscalers | 7.27.0 | 7.46.0 | 3.6.8 | 1.16.0 |
```

Comment:
❓ For resources collected by the Cluster Agent, why do we say the minimal agent version is `7.27`. Shouldn't it match the version which is released at the same time as the cluster agent to reduce the confusion?